hs/src/Reach/Parser/Common.hs

```diff
@@ -0,0 +1,8 @@
+module Reach.Parser.Common
+  ( ParserOpts(..)
+  ) where
+
+data ParserOpts = ParserOpts
+  { dirDotReach :: FilePath
+  , canGit      :: Bool
+  }
```

Comment:
This should just be in `Parser`

Reply:
We can't because that'd cause a cyclic dependency graph.